README.md

```diff
@@ -96,6 +96,10 @@ A getter for abstracting the underlying nbt, get's the mob name from a spawn egg
 
 ## History
 
+### 1.12.3
+* [add commands workflow (#99)](https://github.com/PrismarineJS/prismarine-item/commit/a4f5d593be8a80fbe539369efe2d1127c9785583) (thanks @extremeheat)
+* [Update to node 18.0.0 (#98)](https://github.com/PrismarineJS/prismarine-item/commit/52098ca2b0a9eafa6bcde3cdfb3c2315bab258a5) (thanks @Romain Beaumont)
```

Comment:
A detail but wonder if it's possible to replace the name by the pseudo in the thanks, so that @ is actually working

Reply:
yeah, the action can switch to using the Github API for this (currently it's looking at local git commit logs which don't store github usernames)

Reply:
fixed